.github/workflows/e2e_ios.yml

```diff
@@ -16,9 +20,20 @@ env:
   APP_CLIENT_SECRET: ${{ secrets.APP_CLIENT_SECRET }}
   # IAM production app client secret
   APP_CLIENT_SECRET_PROD: ${{ secrets.APP_CLIENT_SECRET_PROD }}
+  # Slack API token used by fastlane
+  SLACK_API_TOKEN: ${{ secrets.SLACK_API_TOKEN }}
```

Comment:
@Sparowhawk this stuff should be in a separate ticket/PR, we're not alerting Slack yet until all the runs pass. Can you break it out? 

Also shouldn't be any iOS changes in this PR since it's an android ticket

Reply:
removed the iOS changes